app/Http/Controllers/Bible/TextControllerV2.php

```diff
@@ -34,7 +34,7 @@ class TextControllerV2 extends APIController
      * API Note: I removed the v4 openapi docs. Returning text for a fileset/book/chapter is now handled in BibleFileSetsController:showChapter, along with all other filesets
      *
      * @OA\Get(
-     *     path="/text/verse",
```

Comment:
It has been changed because we have two paths with the same value and swagger currently doesn't support to merge the two identical paths despite they have a different operationId. Let me know if it is ok or we need to search another solution.

Http/Controllers/Bible/TextController.php line 49.

Reply:
@vichugofsl /text/verse is only for v2. Please remove it from TextController.php at line 49

Reply:
@bradflood I has been updated and I have changed the path into the TextController.php file at line 49